language/vm/vm-runtime/src/libra_vm.rs

```diff
@@ -228,6 +229,9 @@ impl LibraVM {
         state_view: &dyn StateView,
         remote_cache: &dyn RemoteCache,
     ) -> VMResult<VerifiedTranscationPayload> {
+        if transaction.sender() == core_code_address() {
+            return Err(VMStatus::new(StatusCode::NULL_SENDER));
```

Comment:
Does this need a special case? The prologue code that checks whether the sender's account exists ought to cover this, right?

Reply:
I was just being extra cautious there. I agree that this check would be covered by the prologue check but since this is just a one line check I'd love to add this in due to the special privilege of 0x0.